docs/index.asciidoc

```diff
@@ -70,6 +70,49 @@ Events will be tagged with `_geoip_expired_database` tag to facilitate the handl
 
 TIP: When possible, allow Logstash to access the internet to download databases so that they are always up-to-date.
 
+[id="plugins-{type}s-{plugin}-manage_update"]
+==== Manage your own database updates
+
+**Use a proxy endpoint**
+
+If you can't connect directly to the Elastic GeoIP endpoint, consider setting up
+a secure proxy. You can then specify the proxy endpoint URL in the
+`xpack.geoip.download.endpoint` setting in `logstash.yml` file.
+
+**Use a custom endpoint**
+
+If you work in air-gapped environment and can't update your databases from the Elastic endpoint,
+you can bootstrap the environment as follows.
+
+You can create a service that mimics the Elastic GeoIP endpoint. You can then
+get automatic updates from this service.
+
+. Download your `.mmdb` database files from the
+http://dev.maxmind.com/geoip/geoip2/geolite2[MaxMind site].
+
+. Copy your database files to a single directory.
+
+. Download {es} from https://www.elastic.co/downloads/elasticsearch[Elastic]
```

Comment:
```suggestion
. https://www.elastic.co/downloads/elasticsearch[Download {es}].
```

Reply:
Another option: 

. Download {es} from the https://www.elastic.co/downloads/elasticsearch[Elasticsearch download page].